docs/src/main/asciidoc/_developer-guide/JPA_Reference_Guide.adoc

```diff
@@ -858,6 +858,12 @@ persistence context is closed (e.g. business method ends). Defaults to
 false (entities are cleared after EntityManager invocation) and if set
 to true, the detach is deferred until the context is closed.
 
+|wildfly.jpa.skipquerydetach |Controls whether transaction scoped
+persistence context used in non-JTA transaction thread, will detach
```

Comment:
s/used in/used in a/g
s/thread,/thread/g

Reply:
will update jboss.as.jpa.deferdetach also for this change.